README.md

```diff
@@ -86,7 +86,8 @@ their use case. (Hint: it might, or might not, be pychubby!)
 | Feature/Need                | pychubby                        | pex                                  | zipapp                         | PyInstaller / PyOxidizer                         |
```

Comment:
One notable missing tool is [`shiv`](https://github.com/linkedin/shiv) which grew out of an ex-Pex user wanting something simpler - no cross-platform support, etc, at Linked In.